docs/user_guide/fairness.rst

```diff
@@ -1,125 +1,49 @@
-underrepresented parts of the data, with the goal of becoming more relevant in decision-making computations.
-
-As such, the aim of Fairlens is to analyze datasets and produce fairness scores in the absence of a machine learning model,
-picking up discrepancies between different sensitive demographics with respect to a target column, usually representing
-a score or classification.
 
-Types of Biases
----------------
+  P(R \mid A = a) = P(R)\quad \forall a \in A
 
-Some of the predominant causes of unfairness resulting from structural datasets are highlighted in the
-works of Kamishima et al. on a regularization approach to fairness in models [1]. In summary, the
-following are ways in which biases can exist in structural data.
-
-Prejudice
-^^^^^^^^^
-
-Prejudice occurs when the target column in a dataset, or any column used in a prediction model, is
-dependant on a sensitive column. There are three types of prejudice.
-
-- Direct prejudice occurs when a sensitive column is directly used in a prediction model therefore the
-  classification results will depend on sensitive features producing direct discrimination.
-- Indirect prejudice is defined as the presence of a statistical dependence between a sensitive feature and
-  the target. This results in a high correlation between the values of the sensitive column and the target.
-  Even if the target is no longer a function of the sensitive data, it is still clearly influenced by it.
-- Latent prejudice is defined as a statistical dependence between a sensitive variable and a non-sensitive one.
-  For example, if there exists a correlation between a sensitive column and a non-sensitive column
-  which is then used by the predictor, we can say the determination is discriminatory, as the
-  dependence is still present, but a layer deeper.
-
-Underestimation
-^^^^^^^^^^^^^^^
+.. math::
+  \text{or}
 
-Underestimation appears when a model is not fully converged due to the size limitations of a training dataset. Assuming
-the presence of a an algorithm that is able to learn without commiting indirect prejudice, it is going to produce a fair
-predictor if the the training examples are infinite.
+.. math::
 
-However, if the size of the data is limited and some groups are underrepresented, the model might unintentionally produce
-unfair results based on the local observations of the sample distributions.
+  P(R \mid A = a) = P(R \mid A = b)\quad \forall a,b \in A
 
-Negative Legacy
-^^^^^^^^^^^^^^^
+While Barocas et. al propose 2 alternative fairness or non-discrimination criteria, separation and sufficiency,
+which are applicable to a range of problems, we have chosen to work with independence because of its generality [2]_.
+Working with this abstract definition of fairness, we quantify the bias of a variable :math:`T` in a group :math:`a`,
+as the statistical distance between the the probability distributions of :math:`P(T \mid A = a)` and :math:`P(T)`.
```

Comment:
I think after this section we need to give more details, and how the features of fairlens relate to this

Reply:
Adding another section for this.